File: postkit/com.wapo.rainbow.article.model/src/main/java/com/wapo/flagship/json/AdSetConfig.java
package com.wapo.flagship.json;

import java.io.Serializable;

public class AdSetConfig implements Serializable {

    private String adSetZone;
    private AdSetUrls adSetUrls;

    public String getAdSetZone() {
        return adSetZone;
    }

    public AdSetUrls getAdSetUrls() {
        return adSetUrls;
    }
}

File: android-tablet/src/main/java/com/wapo/flagship/features/articles2/models/deserialized/video/AdSetConfig.kt
package com.wapo.flagship.features.articles2.models.deserialized.video

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class AdSetConfig(
    @Json(name = "adSetUrls")
    val adSetUrls: AdSetUrls?,
    @Json(name = "adSetZone")
    val adSetZone: String?,
)
